manifests/j/JakobDev/jdTextEdit/10.3/JakobDev.jdTextEdit.installer.yaml

```diff
@@ -0,0 +1,19 @@
+PackageIdentifier: JakobDev.jdTextEdit
+PackageVersion: 10.3
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
The package installs normally, but needs the dependency to run.
![image](https://user-images.githubusercontent.com/114952565/214371470-580780ac-051d-480b-8fce-c891997038e2.png)

Reply:
The Bot says, that dependencies are currently not supported. Should I just upload a new Installer that has everything included?

Reply:
Yes, bundling dependencies into your installer is currently the recommended method. Dependency support is becoming more robust as newer versions are released, but might still be experimental in some situations.

Reply:
I have now updated the Installer. It should now work without dependencies.